cpp/example_code/dynamodb/dynamodb_getting_started_scenario.cpp

```diff
@@ -0,0 +1,914 @@
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+/**
+ * Before running this C++ code example, set up your development environment, including your credentials.
+ *
+ * For more information, see the following documentation topic:
+ *
+ * https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/getting-started.html
+ *
+ * Purpose
+ *
+ * Demonstrates using the AWS SDK for C++ to create an Amazon DynamoDB table and
+ *  and perform a series of operations on the table.
+ *
+ * 1. Create a table with partition: year (N) and sort: title (S). (CreateTable)
+ * 2. Add a new movie. (PutItem)
+ * 3. Update the rating and plot of the movie by using an update expression.
+ *    (UpdateItem with UpdateExpression + ExpressionAttributeValues args)
+ * 4. Put movies in the table from moviedata.json--download it from the DynamoDB guide. (BatchWriteItem)
```

Comment:
Is it the Developer Guide that's being referenced here, or another one? 

 * 4. Put movies in the table from moviedata.json, downloaded from the Amazon DynamoDB Developer Guide. (BatchWriteItem)